app/views/data_access/retrieval.html.erb

```diff
@@ -14,9 +14,9 @@
   <nav class="data-access-nav">
     <a href="<%=edsc_path "/data/configure?#{@retrieval.source}" %>"><i class="fa fa-arrow-circle-o-left"></i> Back to Data Access Options</a>
   </nav>
-  <header class="data-access-header">
-    <h1>Success!</h1>
-    <p>Your request has been processed.  See below for information on accessing your data.</p>
+  <header class="data-access-header" data-bind="visible: ui.projectList.collectionsToDownload().length == 0">
+    <h1>Request in Progress</h1>
+    <p>Your request is being processed.  Results will appear shortly.</p>
```

Comment:
For some reason this silly github won't let me reply to each of your comments :)

Nice catch on the instances of it not disappearing; it must be logically based upon only the 'success' block.  I can look at that as well as adding a bit of padding for the data access container.

I'll put a WIP flag on this and then rework it a bit.